hackathon_site/dashboard/frontend/src/pages/Cart/Cart.tsx

```diff
@@ -71,8 +70,12 @@ const Cart = () => {
                             style={{ width: "100%", marginTop: "10%" }}
                             data-testid="cart-linear-progress"
                         />
+                    ) : cartQuantity === 0 ? (
+                        <Typography variant="subtitle2" align="center" paragraph>
+                            NO ITEMS IN CART
+                        </Typography>
                     ) : (
-                        mockCartItems.map(({ hardware_id, quantity }, i) => {
+                        cartItems.map(({ hardware_id, quantity }, i) => {
                             return (
                                 <div key={i}>
                                     <CartCard
```

Comment:
just curious, is there a missing ">" or it is the syntax rule in redux?

Reply:
It does have a closing > but I think since the properties of this component didn't change GitHub isn't showing the differences
CartCard has some props being passed in, they're the same as what's on [https://www.github.com/ieeeuoft/hackathon-template/tree/develop/hackathon_site%2Fdashboard%2Ffrontend%2Fsrc%2Fpages%2FCart%2FCart.tsx](develop)

Reply:
Also not a Redux thing, this is regular old React JSX syntax. The closing `/>` is a few lines down.